package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "revai-node-sdk",
-  "version": "2.6.3",
+  "version": "3.0.0",
```

Comment:
@amikofalvy @kbridbur I pushed it to a major version since the change would affect customers who may have been submitting to the stream objects other than buffers (they shouldn't be but just in case)